Sources/Extensions/UIViewController+Hero.swift

```diff
@@ -324,7 +324,9 @@ public extension HeroExtension where Base: UIViewController {
             parentVC.present(next, animated: false, completion: completion)
           }
         } else {
+          #if TARGET_IS_EXTENSION
```

Comment:
Maybe should be ?
```suggestion
          #if !TARGET_IS_EXTENSION
```

Reply:
No, this is correct. If you are in an extension `UIApplication.shared` in unavailable

Reply:
Exactly, this if is calling the following line using `UIApplication.shared` is it IS an extension.

My change is the inverse, only if it's NOT an extension, unless I'm drunk?

Reply:
This code is problematic either way. I think it's not the correct conditional check. It either always or never disables this block of code despite how I flip the extension setting. Need to dig into a better solution. Supringly hard to find. 

I made a new PR to solve other issues #681